src/plugins/plot/src/configuration/PlotSeries.js

```diff
@@ -377,13 +377,17 @@ define([
          * @public
          */
         updateFiltersAndRefresh: function (updatedFilters) {
-            this.filters = updatedFilters;
-            this.reset();
-            if (this.unsubscribe) {
-                this.unsubscribe();
-                delete this.unsubscribe;
+            if (this.filters && !_.isEqual(this.filters, updatedFilters)) {
```

Comment:
This should likely be part of a different branch and PR? I don't think it's specific to Conditionals so if it's required to go out sooner, it could.

Reply:
See previous comment re. Flex Layout